### DISCUSSION TITLE
23.10.04 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#17] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - Spotify 소셜 로그인 과정 학습 및 구현 (진행 중)
- 내일은 어떤 일을 하나요?
    - 소셜 로그인 구현

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - Spring Event로 노래 추가 시 event, listener 구현
  - SMTP 기반 메일 sender 구현

- 내일은 어떤 일을 하나요?
  - 노래 추가 외에도 다양한 알림에 대응할 수 있도록 기능 추가
  - 테스트 코드 상세화

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * Jacoco 적용
  * 테스트 자동화 적용
* 내일은 어떤 일을 하나요?
  * 프로젝트 설정 변경
  * 주문 도메인 생성

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 로컬 개발 환경 구축
    - 깃허브 Project 관리 Issue 등록
    - 플레이리스트 엔티티 등록
    - 기능 구현과 관련된 테스트 코드 작성 학습

- 내일은 어떤 일을 하나요?
    - 플레이리스트 생성, 조회, 수정, 삭제 기능 구현
    - 기능 구현과 관련된 테스트 코드 작성
